src/overlays/actors/ovl_En_Bigpamet/z_en_bigpamet.h

```diff
@@ -23,11 +23,11 @@ typedef struct {
 typedef struct EnBigpamet {
     /* 0x000 */ Actor actor;
     /* 0x144 */ SkelAnime skelAnime2;
-    /* 0x188 */ Vec3s jointTable2[OBJECT_TL_2_LIMB_MAX];
-    /* 0x1D6 */ Vec3s morphTable2[OBJECT_TL_2_LIMB_MAX];
+    /* 0x188 */ Vec3s jointTable2[SNAPPER_LIMB_MAX];
+    /* 0x1D6 */ Vec3s morphTable2[SNAPPER_LIMB_MAX];
     /* 0x224 */ SkelAnime skelAnime1;
-    /* 0x268 */ Vec3s jointTable1[OBJECT_TL_1_LIMB_MAX];
-    /* 0x280 */ Vec3s morphTable1[OBJECT_TL_1_LIMB_MAX];
+    /* 0x268 */ Vec3s jointTable1[SPIKED_SNAPPER_LIMB_MAX];
+    /* 0x280 */ Vec3s morphTable1[SPIKED_SNAPPER_LIMB_MAX];
```

Comment:
Did you want to name these two joint/morph tables `snapperJointTable`/`spikedSnapperJointTable`? Same for `kame.h`?

Reply:
Was going to do it later, but sure, let me do it now.